### PR TITLE
e2e: run cleanup script before running e2e tests

### PR DESCRIPTION
**Pull request by Fraccaman**
## Checklist before merging 
- [ ] If this PR has some consensus breaking changes, I added the corresponding `breaking::` labels
    - This will require 2 reviewers to approve the changes
- [ ] If this PR requires changes to the docs or specs, a corresponding PR is opened in the `namada-docs` repo
    - Relevant PR if applies: 
- [ ] If this PR affects services such as `namada-indexer` or `namada-masp-indexer`, a corresponding PR is opened in that repo
    - Relevant PR if applies:

### CODE REVIEW

**Comment by Fraccaman** [comment]
can't work :(

**Comment by tzemanovic** [comment]
> can't work :(

it looks like because of `pkill -9 -f namada*`? Let's just kill gaiad

File: .github/workflows/scripts/pre-e2e-test.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+pkill -9 -f gaia || true
+pkill -9 -f namada* || true
+pkill -9 -f hermes || true
```
Comment (tzemanovic) [inline .github/workflows/scripts/pre-e2e-test.sh:3]:
```suggestion
pkill -9 -f gaiad || true
pkill -9 -f hermes || true
```

**Comment by Fraccaman** [comment]
this doesn't work :( nextest runs the script as a setup script, not after every test :(

**Pull request closed without merging**